torch/testing/_internal/distributed/rpc/rpc_test.py

```diff
@@ -833,8 +828,11 @@ def test_py_raise_in_user_func(self):
 
     @dist_init
     def test_script_function_exception(self):
-        dst_rank = (self.rank + 1) % self.world_size
-
+        @torch.jit.script
```

Comment:
torchscript functions does no need to be top-level ones? Because they are looked up in cu instead of using interpreter?

Reply:
I think so. `@torch.jit.script` decorator register them in `cu`. So Python global name lookup requirment does not apply here.